README.md

```diff
@@ -102,3 +102,56 @@ vault:
 ```
 
 - After successful start, you shall require entering a new password as provided in vault.
+
+### Auth0
```

Comment:
I'm not sure we need this since we don't have it for Okta. I think we should point to the [existing documentation for OIDC](https://www.jhipster.tech/security/#oauth2) or add Okta here.

Reply:
Great idea ! I added a section for securing the JHipster Register with OAuth authentication. We point to the existing documentation for OIDC